packages/metrics/tests/e2e/decorator.test.MyFunction.ts

```diff
@@ -2,7 +2,7 @@ import { Metrics, MetricUnits } from '../../src';
 import { Context } from 'aws-lambda';
 import { LambdaInterface } from '../../examples/utils/lambda/LambdaInterface';
 
-const namespace = process.env.EXPECTED_NAMESPACE ?? 'CDKExample';
+const namespace = process.env.EXPECTED_NAMESPACE ?? 'LambdaPowertoolsTypeScript-CDKExample';
```

Comment:
Any particular reason why are we applying the same change to e2e tests? I'm open to discuss the change but in a separate PR, so to keep this one focused on examples.

Reply:
Because the Metric examples are found in Metrics > All metrics > CdkExample.
Better to have a more specific name for users that may have other CDK stacks in their account, though maybe unnecesary to change in tests... couldn't find where else CdkExample was coming from.

Reply:
Yea agree that we should find a more specific name for these metrics' namespace but this stack in the tests is completely unrelated from the one in examples so I'd be more keen to make this change in another PR. In any case happy to hear also the opinion of a second reviewer, let's leave the conversation open.